chemked/tests/test_validation.py

```diff
@@ -119,6 +119,31 @@ def test_invalid_DOI(self):
         v.validate({'reference': {'doi': '10.1000/invalid.doi'}}, update=True)
         assert v.errors['reference'] == 'DOI not found'
 
+    def test_invalid_ORCID(self):
+        """Test for proper response to incorrect/invalid ORCID.
+        """
+        v.validate({'file-author': {'ORCID': '0000-0000-0000-0000',
+                   'name': 'Kyle Niemeyer'}}, update=True
+                   )
+        assert v.errors['file-author'] == 'ORCID incorrect or invalid for Kyle Niemeyer'
+
+    def test_invalid_ORCID_name(self):
+        """Test for proper response to incorrect name with ORCID.
+        """
+        v.validate({'file-author': {'ORCID': '0000-0003-4425-7097',
+                   'name': 'Bryan Weber'}}, update=True
+                   )
+        assert v.errors['file-author'] == 'name incorrect, should be Kyle Niemeyer'
```

Comment:
I think this error should be `Name and ORCID do not match. Name supplied: Bryan Weber. Name associated with ORCID: Kyle Niemeyer` or something similar. It could also be that the ORCID is wrong :smile: